app-editors/vis/vis-9999.ebuild

```diff
@@ -14,16 +14,18 @@ if [ "${PV}" == "9999" ]; then
 else
 	SRC_URI="
 		https://github.com/martanne/vis/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
-		test? ( https://github.com/martanne/vis-test/archive/${VIS_TEST_COMMIT}.tar.gz -> vis-test-${VIS_TEST_COMMIT}.tar.gz )
+		test? ( https://github.com/martanne/vis-test/archive/${VIS_TEST_COMMIT}.tar.gz
+			-> vis-test-${VIS_TEST_COMMIT}.tar.gz
+		 )
 	"
-	KEYWORDS="~amd64 ~arm ~riscv ~x86"
+	KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~x86"
 fi
 
 DESCRIPTION="modern, legacy free, simple yet efficient vim-like editor"
 HOMEPAGE="https://github.com/martanne/vis"
 LICENSE="ISC MIT"
 SLOT="0"
-IUSE="+ncurses +lua selinux test tre"
+IUSE="acl +ncurses +lua selinux test tre"
```

Comment:
Making acl optional means `sys-apps/acl` in `DEPEND` should also be optional, and might make sense to set it as `+acl` in IUSE so it's enabled by default even if the selected profile doesn't.